### PR TITLE
Merge pull request #27 from jwfehr/master

### PR DESCRIPTION
**Pull request by zwbce**
Docugen Comments in movement.c @ElyonsAnointed @jwfehr @zwbce

### CODE REVIEW

**Comment by ghost** [comment]
Diggity ;) +1

**Comment by jwfehr** [comment]
+1

File: movement.c [hunk 1]
```diff
@@ -65,6 +85,14 @@ void turnCounterClockwise(oi_t*sensor, int degrees)
 }
 
 /// Move Forwards Given Number of Centimeters with Sensors Activated
+/**
+ *Moves the robot forward a given number of centimeters with bump and cliff sensors active.
+ *If sensors are activated the robot will stop, back up 2 cm and notify the controller of the sensor status, as well the net distance the robot travelled.
```
Comment (mfborman) [inline movement.c:90]: Generally a good idea to new line long comments

**Comment by mfborman** [comment]
+1

**Pull request merged**